instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/transactions_class_methods.rb

```diff
@@ -19,7 +19,7 @@ class << base
           # Contains ActiveRecord::Transactions::ClassMethods to be patched
           module ClassMethods
             def transaction(**options, &block)
-              tracer.in_span("#{self}.transaction") do
+              tracer.in_span("#{name}.transaction") do
```

Comment:
Do you have a link to where this method is defined?

Reply:
I think this is `Class.name` in Ruby core since this module is a mixin for the Class methods

Reply:
Yeah... part of me wants to do something like 

`Span{name: activerecord.transaction, code.namespace: MyModel`

Reply:
@robertlaurin?

Reply:
>The "namespace" within which `code.function` is defined. Usually the qualified class or module name, such that `code.namespace` + some separator + `code.function` form a unique identifier for the code unit.

From https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/span-general/#source-code-attributes

Seems like the right attribute. 

From a high level of readability in a trace viewer, seeing the more specific "#{Model}.transaction" has some ergonomic properties to it. 

From a general cardinality perspective around span naming, this is less desirable. In practice the amount of models that exist is rather bounded even if there are a lot of them in any particular application.

Where this gets a bit tricky is that the blazer adapter is appending the object id to the adapter name, which basically makes this unbounded cardinality... which is unfortunate for what we're doing here.

I talked with MattAlp about this recently, and I've gone back and forth since then. I think what I would like to see here is to implement Ariels suggestion of 

>Span{name: activerecord.transaction, code.namespace: MyModel}

Long term this is the direction I'd like to see.

Reply:
Wow, what have I done??? 😛 

I think [@arielvalentin's suggestion](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/617/files#r1290451630) makes sense 😄